XcodeServerSDKTests/IntegrationTests.swift

```diff
@@ -23,4 +23,19 @@ class IntegrationTests: XCTestCase {
         self.waitForExpectationsWithTimeout(10, handler: nil)
     }
     
+    // MAKR: Commits
+    func testGetIntegrationCommits() {
+        
+        let exp = self.expectationWithDescription("Network")
+        let server = self.getRecordingXcodeServer("get_integration_commits")
+        server.getIntegrationCommits("56ad016e2e3993ca0b8ed276050150e8") { (commitIntegrations, error) in
+            XCTAssertNil(error, "Error should be nil")
+            
```

Comment:
If you could add a couple of assertions about the passed-in `commitIntegrations`'s values, so that we verify the parsing went well, that'd be great. (If you were already planning to do this, ignore me)